acceptance-tests/journey/src/test/java/uk/gov/di/ipv/cri/kbv/acceptancetest/journey/KBVJourneyTest.java

```diff
@@ -0,0 +1,123 @@
+package uk.gov.di.ipv.cri.kbv.acceptancetest.journey;
+
+import com.gargoylesoftware.htmlunit.Page;
+import com.gargoylesoftware.htmlunit.WebClient;
+import com.gargoylesoftware.htmlunit.html.HtmlButton;
+import com.gargoylesoftware.htmlunit.html.HtmlElement;
+import com.gargoylesoftware.htmlunit.html.HtmlForm;
+import com.gargoylesoftware.htmlunit.html.HtmlPage;
+import com.gargoylesoftware.htmlunit.html.HtmlRadioButtonInput;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.util.Base64;
+import java.util.List;
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+class KBVJourneyTest {
+
+    @DisplayName("answer all questions correctly and produce a VC with a verification score of 2")
+    @Test
+    public void shouldPassKBV() throws Exception {
```

Comment:
no need for `public`

Reply:
I agree. We should do API testing and the tests should live in the same repo as dev.